azuredevops/internal/client/client.go

```diff
@@ -43,6 +45,8 @@ type AggregatedClient struct {
 	GraphClient                   graph.Client
 	V5GraphClient                 v5graph.Client
 	OperationsClient              operations.Client
+	V5PipelinesChecksClient       v5pipelineschecks.Client
```

Comment:
There's no good choice on what version of API to use for this, v5, v6 and probably v7 are all incomplete in some way or another. I started with v5 since it's what the web UI is using and actually has the ["Settings" field](https://github.com/microsoft/azure-devops-go-api/blob/dev/azuredevops/pipelineschecks/models.go#L17) in the CheckConfiguration data model which is [missing](https://github.com/microsoft/azure-devops-go-api/blob/dev/azuredevops/v6/pipelineschecks/models.go#L44) in the v6 api, and based on the API docs it looks like it'll be missing in v7 too if the clients are ever regenerated.